### ISSUE TITLE
linter: OXC extension doesn't show inline errors for custom js-plugin

### ISSUE THREAD

**Issue by yzhe554** [oxc-project/oxc#18269]
### What version of Oxlint are you using?

1.4.1

### What command did you run?

pnpm lint

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
  "$schema": "./node_modules/oxlint/configuration_schema.json",
  "ignorePatterns": [],
  "categories": {
    "correctness": "off",
    "suspicious": "off",
    "pedantic": "off",
    "style": "off"
  },
  "plugins": [],
  "jsPlugins": ["./js-plugins/index.ts"],
  "rules": {
    "test/nextjs-require-prefetch": "error",
  },
  "overrides": [],
  "env": {
    "builtin": true
  }
}

```

### What happened?

Reproducible repo: https://github.com/yzhe554/oxlint-nextjs

I implemented a custom rule using js-plugins to alert when prefetch is missing from nextjs Link component.
The error is correctly reported from CLI but the extension doesn't show it inline.

<img width="642" height="514" alt="Image" src="https://github.com/user-attachments/assets/c6ad06a4-efe3-46af-b4f1-13da780c8aca" />

Please run `pnpm lint` from the repo.

**Comment by connorshea** [comment]
Thank you for reporting and providing a reproduction case! This is a known limitation that we're tracking in https://github.com/oxc-project/oxc/issues/14826, we are close to adding JS Plugin support to editor integrations, but it's not quite done yet.
